### ISSUE TITLE
DO WANT

### ISSUE THREAD

**Issue by EugeneDae** [picturepan2/print.css#1]
![giphy](https://cloud.githubusercontent.com/assets/980835/24836183/7f6fde28-1d1d-11e7-9684-2ed348ce17e7.gif)

**Comment by picturepan2** [comment]
@EugeneDae LOL. It is still drafts on my device. BTW, what features do you need?

**Comment by Anachron** [comment]
Well I'd love this too! 
Black and white print style would be awesome, and also spectre.css should be fully supported (which is obvious I guess).

**Comment by picturepan2** [comment]
@Anachron Yes, I will use blank/white and non-background mode for better and quicker printing, also environment friendly.

**Comment by Anachron** [comment]
Great,- on what timescale do you plan the execution? Do you have the docs somewhere so you can upload them?

I could start with some basics, like "print-only" and "print-hide" classes. 

One would then add "print-hide" to headers and footers and "print-only" to additional information like page author. 

Or should everything be hidden by default and "print-show" and "print-only" exist? 

(Using the print media directive we can easily add it to mix all other styles)

**Comment by picturepan2** [comment]
@Anachron No. I am doing it and I haven't committed any code yet. Thanks.

**Comment by rhukster** [comment]
Looking forward to this also.

**Comment by picturepan2** [comment]
@rhukster What features do you need?
